docs/sources/installation/centos.md

```diff
@@ -6,153 +6,171 @@ page_keywords: Docker, Docker documentation, requirements, linux, centos, epel,
+CentOS documentation.
 
-## Kernel support
+##Prerequisites
 
-Currently the CentOS project will only support Docker when running on kernels
-shipped by the distribution. There are kernel changes which will cause issues
-if one decides to step outside that box and run non-distribution kernel packages.
+Docker requires a 64-bit installation regardless of your CentOS version. 
+Docker requires that your kernel must be 3.10 at minimum. CentOS 7 runs the 3.10 kernel, 6.5 does not. We make an exception for CentOS 6.5. To run Docker on [CentOS-6.5](http://www.centos.org) or later, you need kernel 2.6.32-431 or higher. 
 
-To run Docker on [CentOS-6.5](http://www.centos.org) or later, you will need
-kernel version 2.6.32-431 or higher as this has specific kernel fixes to allow
-Docker to run.
+To check your current kernel version, open a terminal and use `uname -r` to
+display your kernel version:
 
-## CentOS-7
+    $ uname -r 
+    2.6.32-431.el6.x86_64
 
-### Installation
 
-Docker is included by default in the CentOS-Extras repository. To install
-run the following command:
+## Install
 
-    $ sudo yum install docker
+You use the same installation procedure for all versions of CentOS,
+only the package you install differs. There are two packages to choose from:
 
-Please continue with the [Starting the Docker daemon](#starting-the-docker-daemon).
+<table>
+  <tr>
+    <th>Version</th>
+    <th>Package name</th>
+  </tr>
+  <tr>
+    <td>6.5 and  higher</td>
+    <td>
+    <a href="http://get.docker.com/docker/1.7.0/rpms/centos-6/RPMS/x86_64/docker-engine-1.7.0-0.1.el6.x86_64.rpm">
+    http://get.docker.com/docker/1.7.0/rpms/centos-6/RPMS/x86_64/docker-engine-1.7.0-0.1.el6.x86_64.rpm</a>
+    </td>
+  </tr>
+  <tr>
+    <td>7.X</td>
+    <td>
+     <a href="http://get.docker.com/docker/1.7.0/rpms/centos-7/RPMS/x86_64/docker-engine-1.7.0-0.1.el7.centos.x86_64.rpm">
+    http://get.docker.com/docker/1.7.0/rpms/centos-7/RPMS/x86_64/docker-engine-1.7.0-0.1.el7.centos.x86_64.rpm</a>   
+    </td>
+  </tr>
+</table>
 
-### Uninstallation
 
-To uninstall the Docker package:
+This procedure depicts an installation on version 6.5. If you are installing on
+7.X, substitute that package for your installation. 
 
-    $ sudo yum -y remove docker
+1. Log into your machine as a user with `sudo` or `root` privileges.
 
-The above command will not remove images, containers, volumes, or user created
-configuration files on your host. If you wish to delete all images, containers,
-and volumes run the following command:
+3. Make sure your existing packages are up-to-date.
 
-    $ rm -rf /var/lib/docker
+		$ sudo yum update
+		
+2. Download the Docker RPM to the current directory.
+		
+		$ curl -O -sSL http://get.docker.com/docker-engine-1.7.0-0.1.el6.x86_64.rpm
 
-You must delete the user created configuration files manually.
+4. Use `yum` to install the package.
 
-## CentOS-6.5
+		$ sudo yum localinstall --nogpgcheck docker-engine-1.7.0-0.1.el6.x86_64.rpm
```

Comment:
Not too much on the rpm-fu, but based on the `--nogpgcheck`, it looks like we're not verifying the downloaded package?